### PR TITLE
OCPBUG 13560: Added worker nodes related reference link in the additional info

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.12+

Issue: [OCPBUG13560](https://issues.redhat.com/browse/OCPBUGS-13560)

Link to docs preview: [Additional Resources](https://67336--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer#recommended-resources-for-topologies), [Note](https://67336--ocpdocs-pr.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-sno-worker-nodes)

QE review:
- [x] QE has approved this change.
- [x] SME has approved this change

Additional information:

### CODE REVIEW

File: installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
Comment (andfasano) [inline installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:28]: AFAIU, the mentioned procedures for adding a worker to an already existing cluster requires an internet connection, so they are not suitable for disconnected environments. Can you please confirm? In such case it could be useful to add a note for such constraint, since the agent-based installer can also be used to deploy a cluster into a disconnected environment

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:28]: @zniu1011 Can you confirm if I have to add the note?

Comment (zniu1011) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:28]: @subhtk We need to add the note. Please see "Requirements for installing single-node OpenShift worker nodes" in https://docs.openshift.com/container-platform/4.12//nodes/nodes/nodes-sno-worker-nodes.html#ai-sno-requirements-for-installing-worker-nodes_add-workers

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:28]: ack.

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:28]: Acknowledged. Let me add the note.

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc]: Hey @zniu1011, can you provide the LGTM label once.

Comment (andfasano) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc]: @zniu1011 IINW the mentioned procedure requires to access requires (at least) to access https://console.redhat.com/openshift to work, so the access to a local registry would not seem to be enough.

Comment (zniu1011) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc]: Ok, thanks @andfasano . @subhtk Can we please make the notes like this: "Please note that the access to OpenShift Cluster Manager or Assisted Installer API is required using the way to add the worker nodes, so it is not suitable for agent-based installed clusters in disconnected environments."

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc]: Sure. Let me change it right away.

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc]: @zniu1011 Done, Please take a look.

Comment (andfasano) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc]: As long as it sounds reasonable to me, I'd like if someone from the assisted team could double-check the phrasing to ensure it matches properly the mentioned case cc @oourfali

File: installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
Comment (oourfali) [inline installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:32]: I'd remove the CLI/API piece.
I'd just say "To add worker nodes, you'll need access to the OpenShift Cluster Manager. As a result, agent-based installed clusters in disconnected environments will not support this method."

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:32]: Ack. let me change it.

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc]: Done.

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc]: @oourfali Please add in the LGTM label.

**Review by dfitzmau — COMMENTED** [review]
Hi @subhtk . Nice work :+1: . I added some minor comments inline.

/remove-label peer-review-in-progress

/label peer-review-done

File: installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
Comment (dfitzmau) [inline installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:30]: The note is not indented so maybe adding a plus symbol will collate the note with the last link?

File: installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
Comment (jeana-redhat) [inline installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:28]: Remove the `.`
```suggestion
* xref:../../nodes/nodes/nodes-sno-worker-nodes.adoc#nodes-sno-worker-nodes[Adding worker nodes to single-node OpenShift clusters]
```

File: installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
Comment (jeana-redhat) [inline installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:34]: This note needs to be relocated.

Additional resources items must contain minimal-length, human-readable text between the square brackets, and no text outside of the square brackets. You can review the guidelines here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

**Review by jeana-redhat — COMMENTED** [review]
Needs a couple fixes before it can merge

/remove-label merge-review-in-progress
/remove-label merge-review-needed

File: installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
Comment (jeana-redhat) [inline installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:28]: Missed this initially, but there is an attribute for SNO:
```suggestion
* xref:../../nodes/nodes/nodes-sno-worker-nodes.adoc#nodes-sno-worker-nodes[Adding worker nodes to {sno} clusters].
```

Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:28]: Ack.

File: installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:34]: Got it. Let me relocate the Note.

File: installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:28]: Ack.

File: installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc
Comment (subhtk) [reply installing/installing_with_agent_based_installer/preparing-to-install-with-agent-based-installer.adoc:34]: I have added the notehere in ["Adding worker nodes to SNO clusters"](https://67336--docspreview.netlify.app/openshift-enterprise/latest/nodes/nodes/nodes-sno-worker-nodes#nodes-sno-worker-nodes) module. @dfitzmau Can you check if it is right?

File: nodes/nodes/nodes-sno-worker-nodes.adoc
Comment (dfitzmau) [inline nodes/nodes/nodes-sno-worker-nodes.adoc:25]: OpenShift Cluster Manage needs to be an attribute as per the common-attributes file.

Comment (subhtk) [reply nodes/nodes/nodes-sno-worker-nodes.adoc:25]: ack.

**Review by dfitzmau — APPROVED** [review]
Hi @subhtk . Peer review: LGTM

**Comment by JoeAldinger** [comment]
@subhtk you have two merge conflicts in 4.13 and 4.12. Ping me when you have the manual CPs ready and I'll merge those, thank you.

**Pull request merged**
